features/step_definitions/monitoring.rb

```diff
@@ -0,0 +1,79 @@
+Then(/^I check replicas of monitoring component for sno cluster$/) do
```

Comment:
@lihongyan1 I think it should be `components` instead since you are checking multiple components here